src/oc/web/actions/org.cljs

```diff
@@ -67,6 +67,23 @@
       (cook/remove-cookie! (router/last-org-cookie)))
     (routing-actions/maybe-404)))
 
+(def default-board "all-posts")
+
+(defn get-default-board [org-data]
+  (let [last-board-slug default-board]
+    ; Replace default-board with the following to go back to the last visited board
+    ; (or (cook/get-cookie (router/last-board-cookie (:slug org-data))) default-board)]
```

Comment:
@bago2k4 commented out?

Reply:
Yeah that's because we want to bring the user always back to AP, we used to bring the user back to the last visited board or AP if nothing was specified in the url.
Want me to remove this code? I think it can still come in handy